src/main/java/nextstep/qna/domain/Question.java

```diff
@@ -1,92 +1,77 @@
 package nextstep.qna.domain;
 
-import nextstep.users.domain.NsUser;
-
 import java.time.LocalDateTime;
 import java.util.ArrayList;
 import java.util.List;
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
 
-public class Question {
-    private Long id;
-
-    private String title;
-
-    private String contents;
-
+public class Question extends BaseEntity {
```

Comment:
```suggestion
public class Question extends DeletableBaseEntity {
```

"이 프로젝트의 정책이 모든 컨텐츠는 삭제는 Soft Delete 정책을 따른다."와 같은 정책이 있다고 상상해 본다면 아래와 같이 부모 객체를 설계하는 것도 한 방법이 될 수 있겠다.

- DeletableBaseEntity는 BaseEntity를 상속
- DeletableBaseEntity가 deleted 상태 값을 가지고 상태 변경 여부 메서드를 가진다.